### PR TITLE
Feature/api design

### PR DESCRIPTION
**Pull request by JainishaParmar**
my pull request

### CODE REVIEW

File: controllers/post.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+exports.getPosts=(req,res)=>{
+    res.json({
+        posts:[{title: "first post"}, {title:"second post"}]
+    });
+    
+}
```
Comment (kapilbarad) [inline controllers/post.js]: @JainishaParmar Please add at the end of line in every file.

CC: @jaimin45 Ashish

File: controllers/post.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+exports.getPosts=(req,res)=>{
+    res.json({
+        posts:[{title: "first post"}, {title:"second post"}]
+    });
```
Comment (JainishaParmar) [inline controllers/post.js:4]: f is capital

File: package.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    "dev": "nodemon index.js"
+  },
+  "author": "jainisha",
+  "license": "ISC",
+  "dependencies": {
+    "date-and-time": "^2.0.1",
+    "express": "^4.17.1",
+    "mongoose": "^6.0.14",
+    "morgan": "^1.10.0",
+    "nodemon": "^2.0.15",
```
Comment (kapilbarad) [inline package.json]: `nodemon` should be developer dependency.

```
npm i nodemon --save-dev
```

CC: @jaimin45 Ashish

File: routes/post.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+const express=require('express')
+const postController=require('../controllers/post')
+
+const router =express.Router()
+router.get('/',postController.getPosts);
+// exports.getPosts=(req,res)=>{
```
Comment (kapilbarad) [inline routes/post.js]: Please remove commented code.

File: swagger.json [hunk 5]
```diff
@@ -0,0 +1,143 @@
+{
+    "swagger": "2.0",
```
Comment (kapilbarad) [inline swagger.json:2]: We need to design APIs of tutorials as I've requested in slack.

**Review by kapilbarad — CHANGES_REQUESTED** [review]

**Review by kapilbarad — CHANGES_REQUESTED** [review]
Some changes are pending.

**Pull request closed without merging**